content/developer/howtos/localization.rst

```diff
@@ -0,0 +1,374 @@
+For example, imagine you want to create a financial report having multiple lines but you have no way to find a rule to dispatch the accounts according their ``code`` or ``name``.
+The solution is the usage of tags, one for each report line, to filter accounts like you want.
+
+Put the tags in `data/account_account_tag_data.xml` file.
+
+.. example::
+  `addons/l10n_lt/data/account.account.template.csv <https://github.com/odoo/odoo/blob/15.0/addons/l10n_lt/data/account.account.template.csv>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_lt/data/account.account.template.csv
+    :language: csv
+    :end-at: account_account_template_1201  
+
+.. example::
+  `addons/l10n_at/data/account_account_template.xml <https://github.com/odoo/odoo/blob/15.0/addons/l10n_at/data/account_account_template.xml>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_at/data/account_account_template.xml
+    :language: xml
+    :start-at: chart_at_template_0010
+    :end-at: </record>
+
+Accounts
+--------
+
+.. seealso::
+   - :ref:`Account References <reference/account_account>`
+   - :doc:`/applications/finance/accounting/payables`
+   - :doc:`/applications/finance/accounting/receivables`
+
+Obviously, *Chart of Accounts* cannot exist without *Accounts*. You need to specify them in `data/account.account.template.csv`.
+
+.. example::
+  `addons/l10n_ch/data/account.account.template.csv <https://github.com/odoo/odoo/blob/15.0/addons/l10n_ch/data/account.account.template.csv>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_ch/data/account.account.template.csv
+    :language: csv
+    :end-at: ch_coa_1171
+
+CSV is prefered but you may use XML format instead.
+
+.. example::
+  `addons/l10n_at/data/account_account_template.xml <https://github.com/odoo/odoo/blob/15.0/addons/l10n_at/data/account_account_template.xml>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_at/data/account_account_template.xml
+    :language: xml
+    :start-at: chart_at_template_0010
+    :end-at: </record>
+
+.. warning::
+
+    Avoid the usage of liquidity ``account.account.type``!
+    Indeed, the bank & cash accounts are created directly at the installation of the localization module and then, are linked to an ``account.journal``.
+
+.. warning::
+
+    Only one account of type payable/receivable is enough.
+
+.. warning::
+
+    Don't create too much accounts: 200-300 is enough.
+
+Next settings for the chart of accounts are set in a separate file, because we need to provide `list of accounts <#accounts>`__ first. In `data/account_chart_post_data.xml` we set some default accounts:
+
+.. todo add reference to account_id in CoA
+
+.. example::
+  `addons/l10n_ch/data/l10n_ch_chart_post_data.xml <https://github.com/odoo/odoo/blob/15.0/addons/l10n_ch/data/l10n_ch_chart_post_data.xml>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_ch/data/l10n_ch_chart_post_data.xml
+    :language: xml
+    :start-at: l10nch_chart_template
+    :end-at: </record>
```

Comment:
The example gives 7 fields only, while it's 21 fields

```xml

  <!-- Receivable/payable accounts. -->
  <field name="property_account_receivable_id" ref="..."/>
  <field name="property_account_payable_id" ref="..."/>

  <!-- Default Income/Expense Account for a Product Template. -->
  <field name="property_account_income_id" ref="..."/>
  <field name="property_account_expense_id" ref="..."/>

  <!-- Default Income/Expense Account for a Product Category. -->
  <field name="property_account_income_categ_id" ref="..."/>
  <field name="property_account_expense_categ_id" ref="..."/>

  <!-- Cash loss/gain accounts. They correspond profit_account_id / loss_account_id fields -->
  <field name="default_cash_difference_income_account_id" ref="..."/>
  <field name="default_cash_difference_expense_account_id" ref="..."/>

  <!-- Loss/gain exchange rate accounts. -->
  <field name="income_currency_exchange_account_id" ref="..."/>
  <field name="expense_currency_exchange_account_id" ref="..."/>

  <!-- Stock valuation accounts. -->
  <field name="property_stock_account_input_categ_id" ref="..."/>
  <field name="property_stock_account_output_categ_id" ref="..."/>
  <field name="property_stock_valuation_account_id" ref="..."/>

  <!-- Tax accounts to  balance current/advance tax payments. -->
  <!-- It's used in `env['account.generic.tax.report']._generate_tax_closing_entries` -->
  <field name="property_tax_payable_account_id" ref="..."/>
  <field name="property_tax_receivable_account_id" ref="..."/>
  <field name="property_advance_tax_payment_account_id" ref="..."/>


  <!-- Base Tax Received Account -->
  <!-- Corresponds to `account_cash_basis_base_account_id` field. -->
  <!-- It's used for taxes with `tax_exigibility` equal to `on_payment` -->
  <field name="property_cash_basis_base_account_id" ref="..."/>

  <!-- POS account. -->
  <field name="default_pos_receivable_account_id" ref="..."/>

  <!-- Payment accounts. -->
  <!--
       Suspense account is used to register an unreconiled bank entries created via bank statement.
       Once it's reconciled, the suspense account in `account.move` will be replaced with a proper account.
       Suspense account is an exceptional case when account is supposed to be replaced in the transaction.
       The new account depends on `account.payment` records. There are two cases
       1. `account.payment' exists before reconciliation aka Blue lines in reconciliation wizard
       2. `account.payment' is created on reconciliation aka regular Black lines in reconciliation wizard

       For case n.1 the new account is Outstanding Receipts/Payments account and the workflow is following:

       Initial transactions:
       * Invoice/bill: Payable/Receivable vs Expenses/Sales
       * Payment:             Outstanding vs Payable/Receivable
       After creating bank statement
       * Bank entry:                 Bank vs Suspense
       After reconciliation:
       * Bank entry:                 Bank vs Outstanding

       For case n.2 the new account is Payable/Receivable account from invoice/bill and the workflow is following:

       Initial transactions:
       * Invoice/bill: Payable/Receivable vs Expenses/Sales
       After creating bank statement
       * Bank entry:                 Bank vs Suspense
       After reconciliation:
       * Bank entry:                 Bank vs Payable/Receivable
  -->
  <field name="account_journal_suspense_account_id" ref="..."/>
  <!-- Outstanding Receipts Account -->
  <field name="account_journal_payment_debit_account_id" ref="..."/>
  <!-- Outstanding Payments Account -->
  <field name="account_journal_payment_credit_account_id" ref="..."/>

```

Reply:
# Current version
![image](https://user-images.githubusercontent.com/186131/143901386-1de63d04-cf61-4f60-a2d8-d71d45b4c238.png)

Reply:
Because you can't put them all at the same time, you first need to create the accounts, look below in the HowTo, where the steps are done in the right order

Reply:
You probably misunderstand me. I'm talking about post-data file, not about the first file `account_chart_template_data`

Reply:
You have the reference for that, no need to put an example with every single account set. I'm not even sure it exists.